### PR TITLE
Multiserver: Add argument for timestamping STDOUT

### LINKED ISSUE #3863: Server: Add timestamps to entries
### What feature would you like to see?

It would be nice to have timestamps to see when events happened on the server

### PR DESCRIPTION
**Pull request by josephwhite**
## What is this fixing or adding?

Adds an argument for Multiserver that will add the timestamp used in the webhost when logging to STDOUT.
To resolve #3863. 

## How was this tested?
Ran Multiserver with an without the new parameter. Also checked server log files to verify timestamps weren't being doubled.

## If this makes graphical changes, please attach screenshots.
```powershell
> python MultiServer.py --logtime
[2024-11-27 07:49:19,795] Archipelago (0.5.1) logging initialized on [The OS™️] running Python 3.12.6
[2024-11-27 07:49:19,871] ALTTPWorld Assigned options through option_definitions which is now deprecated. Please use options_dataclass instead.
[2024-11-27 07:49:21,710] Opening file input dialog for Select multiworld data.
[2024-11-27 07:49:24,730] Loading embedded data package for game Super Mario World
[2024-11-27 07:49:24,749] Loaded save file with 0 received items for 1 players
[2024-11-27 07:49:24,994] Hosting game at [VERY COOL PUBLIC IP ADDY] (████████████)
[2024-11-27 07:49:24,995] server listening on [::]:38281
[2024-11-27 07:49:24,995] server listening on 0.0.0.0:38281
```

### CODE REVIEW

File: MultiServer.py [hunk 1]
```diff
@@ -2356,7 +2358,10 @@ def load_server_cert(path: str, cert_key: typing.Optional[str]) -> "ssl.SSLConte
 
 
 async def main(args: argparse.Namespace):
-    Utils.init_logging("Server", loglevel=args.loglevel.lower())
+    Utils.init_logging(name="Server",
+                       loglevel=args.loglevel.lower(),
+                       write_mode="w",
```
[the remark this answers is not in the document]
Comment (josephwhite) [reply MultiServer.py]: Fair. Is there a use case to not use the default argument?

**Review by Exempt-Medic — APPROVED** [review]
Tested changes through running MultiServer and it seemed to work just fine. Searched for any calls to `init_logging` that might have been broken by this change and did not find any

**Comment by NewSoupVi** [comment]
Super minor thing, but down to the millisecond seems a bit wasteful of space to me, I think down to the second is probably good enough? Not blocking over that but maybe you have an opinion on it

**Comment by Exempt-Medic** [comment]
> Super minor thing, but down to the millisecond seems a bit wasteful of space to me, I think down to the second is probably good enough? Not blocking over that but maybe you have an opinion on it

That's the way it's shown in the console on webhost 🤷

**Comment by NewSoupVi** [comment]
Oh yeah. Lol

**Pull request merged**